mocks/v0/resources/http%3A%2F%2Feda234a4-485f-4c0c-806d-1c9748994c00.com/comments/7e6395b0-01c9-11e8-952c-85c7cdb92fcc/OPTIONS.mock

```diff
@@ -1,4 +1,5 @@
 HTTP/1.1 204 OK
 Content-Type: application/json; charset=utf-8
 Access-Control-Allow-Origin: *
-Access-Control-Allow-Headers: Authorization
\ No newline at end of file
+Access-Control-Allow-Headers: Authorization,Content-Type
+Access-Control-Allow-Methods: GET,PUT,OPTIONS
```

Comment:
No POST?

Maybe we don't need POST because we're auto-sending POST on loading the resources? I'm not sure where all those empty resource are created and whether we want that behaviour. I would say that a resource should only be created when its first comment is created. This might be slightly more complicated on the client side but we won't be barraged by bogus POST events when someone loads a bunch of new resources in some UI somewhere. This might be worth discussing offline.